pallets/parachain-staking/src/lib.rs

```diff
@@ -1869,12 +1722,17 @@ pub mod pallet {
 		/// Join the set of collator candidates
 		pub fn join_candidates(
 			origin: OriginFor<T>,
-			bond: BalanceOf<T>,
+			bond: Balance,
+			liquidity_token: TokenId,
 			candidate_count: u32,
 		) -> DispatchResultWithPostInfo {
 			let acc = ensure_signed(origin)?;
 			ensure!(!Self::is_candidate(&acc), Error::<T>::CandidateExists);
 			ensure!(!Self::is_delegator(&acc), Error::<T>::DelegatorExists);
+			ensure!(
+				<StakingLiquidityTokens<T>>::get().contains_key(&liquidity_token),
```

Comment:
Can you please point me to implementation of `StakingLiquidityTokens` ?

Reply:
https://github.com/mangata-finance/moonbeam/blob/9ce23c12a4a6a7e16fbbdd71a432d2ebd172451e/pallets/parachain-staking/src/lib.rs#L1441-L1448

It is a storage item that stores a BTreeMap